testcases/test_civo_marketplace_integration

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+. "../utils/constants"
+
+
+CLUSTERNAME=marketplace-test
+echo "creating cluster ${CLUSTERNAME}"
+civo k8s create ${CLUSTERNAME} --size=g4c.kube.small --nodes=2 --applications=traefik2-nodeport,civo-cluster-autoscaler,metrics-server,grapple-solution-framework && while ! civo k8s config ${CLUSTERNAME} --save --switch >/dev/null 2>&1; do echo -n "."; sleep 5; done
+
+echo ""
+echo "waiting for grapple to be ready"
+while ! kubectl get -n grpl-system configuration.pkg.crossplane.io grpl 2>/dev/null; do echo -n "."; sleep 2; done && \
+  sleep 5 && \
+  kubectl wait -n grpl-system configuration.pkg.crossplane.io grpl --for condition=Healthy=True --timeout=300s
+
+echo ""
+echo "deploying example"
+grpl e d --GRAS_TEMPLATE=$DB_MYSQL_MODEL_BASED --DB_TYPE=$EXTERNAL_DB
+
+echo "waiting for example to be ready"
+kubectl rollout status -n grpl-mdl-ext deploy grpl-mdl-ext-gras-mysql-grapi grpl-mdl-ext-gras-mysql-gruim
```

Comment:
can you please use the "disc-ext" example to verify the success of the deployment.
thanks